TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/SmartController.java

```diff
@@ -88,11 +88,11 @@ public double getRightTrigger() {
     }
 
     public boolean isLeftTriggerPressed() {
-        return gamepad.left_trigger > 0.3;
+        return gamepad.left_trigger > 0.9;
```

Comment:
i think you can create constant for this magic number, like TRIGGER_PRESSED_THRESHOLD_VALUE, so if 0.9 doesnt work you wont need to change every function, just the variable